apps/chat_web/lib/chat_web/live/chat_live.html.heex

```diff
@@ -7,9 +7,14 @@
         class={"mb-4 py-4 px-4 border rounded-xl #{if message.sender == :bot, do: "bg-gray-200"}"}
       >
         <div class="text-gray-800 line-clamp-2 break-words">
-          <%= message.text 
-          |> ChatWeb.Markdown.to_html()
-          |> raw() %>
+          <%=text = if message.sender == :user do
+                Regex.replace(~r/\n/m, message.text, "\n\n")
```

Comment:
Jedyne czego jestem tutaj niepewny to czy nie zamieniać newlineów na `"  \n"` (nowa linia w markdownie) zamiast `"\n\n"` (nowy paragraf).

Reply:
sprawdze i dam znac

Reply:
o faktycznie dziala, to poprawie